web-jehlomat/src/Components/Inputs/DateTimeInput/DateInput.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, InputHTMLAttributes, useState } from 'react';
+import { default as MInput } from '@mui/material/TextField';
+import styled from '@emotion/styled';
+import DateAdapter from '@mui/lab/AdapterDayjs';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import MobileDatePicker from '@mui/lab/MobileDatePicker';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
```

Comment:
Using `InputHTMLAttributes` isn't definitely wrong, sometime I'm using it too, but just to extend knowledge, there are a different ways https://react-typescript-cheatsheet.netlify.app/docs/advanced/patterns_by_usecase/.